<launch>
	<node pkg="x52_joyext" type="x52_mfd_writer_node" name="x52_mfd_writer_string" output="screen"> 
			<param name="input_type" value="11"/>
			<param name="line" value="1"/>
			<param name="pos" value="0"/>
			<param name="field_length" value="16"/>
			<param name="align" value="1"/>
			<param name="positive_oversize" value="++EE++"/>
			<param name="negative_oversize" value="--EE--"/>
			<param name="axis_or_button" value="false"/>
			<param name="axis_button" value="4"/>
			<param name="stringprint" value="true"/>
			<param name="stringprint_setup" value="Green|-0.5|Yellow|0.5|Red"/>	
			<param name="char_as_int" value="0"/>	
			
			<!-- Remap Topics -->	
			<remap from="~/in" to="/joy" />
			<remap from="~/mfd_text" to="/x52/mfd_text" />
	</node>
</launch>